client/blocks/reader-post-card/index.jsx

```diff
@@ -54,8 +57,69 @@ export default class RefreshPostCard extends PureComponent {
 		onCommentClick: noop
 	};
 
+	constructor( props ) {
+		super( props );
+		[
```

Comment:
**FFTI**: there is a cooler way of binding nowadays

Using this built-in babel transform: https://babeljs.io/docs/plugins/transform-class-properties/
You can write this and it will be bound for you:

`propagateCardClick = () => {...}`